torch/lib/c10d/reducer.cpp

```diff
@@ -566,7 +567,7 @@ void Reducer::mark_variable_ready(VariableIndex index) {
       variable_index < variable_locators_.size(),
       "Out of range variable index.");
   backward_stats_[replica_index][variable_index] =
-      current_time_in_nanos() - backward_stats_base_;
+      current_time_in_nanos() - cpu_timer_.backward_compute_start_time;
```

Comment:
Do we need nanosecond level precision? The autograd profiler reports times in microseconds which should be sufficient?

Reply:
it should be no harm to use nanosecond, especially for small program that runs very fast